### ISSUE TITLE
after logout webhook

### ISSUE THREAD

**Issue by Karitham** [ory/kratos#2771]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/kratos/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/kratos/blob/master/CONTRIBUTING.md).
- [X] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe your problem

I use browser flows for the front end (SPA).

I am caching the Ory Session Cookies on my backend because hitting ory cloud each time is too expensive.

To revoke this cache, I need to know when a user logs out.

### Describe your ideal solution

An after logout webhook (exactly like login except it's a logout), containing enough data to identify the logged-out session (containing the now-revoked token for example).

### Workarounds or alternatives

Hitting my backend to revoke the cache, as well as make a request to ory to delete the session.

Unfortunately, it seems the cookie session tokens are not *session tokens* as described by the API, thus we cannot revoke them by using API flows.

Another alternative would be to emulate a browser flow, but that becomes ridiculous really fast.

### Version

Cloud

### Additional Context

Other session providers such as Auth0 explicitly use this method to make their clouds offering usable.

Without proper callbacks, using browser flows is essentially impossible, and using API flows is just more complex.

**Comment by aeneasr** [comment]
That’s a great idea! Contributions as always welcomed :)

**Comment by maipal-oscoy** [comment]
@aeneasr is this on roadmap?

**Comment by trautonen** [comment]
This would be very useful for example for disabling push notifications for users that have logged out. Any traction for the implementation?

**Comment by getlarge** [comment]
@trautonen Yes, there is. I am pretty interested in contributing to this one.

**Comment by icamys** [comment]
Hi @aeneasr! I'd like to implement this feature. According to the contribution guidelines, we should ask the maintainers first before implementing anything. I wonder if there's any work already in the roadmap and whether a PR is still welcome.
